main/src/ca/uwaterloo/flix/api/lsp/provider/CodeActionProvider.scala

```diff
@@ -43,8 +44,11 @@ object CodeActionProvider {
       else
         Nil
 
-    case ResolutionError.UndefinedName(qn, _, env, _, loc) if overlaps(range, loc) =>
-      mkNewDef(qn.ident.name, uri) :: {
+//    case ResolutionError.UndefinedJvmClass(name, _, loc) if onSameLine(range, loc) =>
```

Comment:
Is this for a followup PR?

Reply:
Yeah, we'll need to first extend UndefinedJvmClass and then complete the code, as discussed above .

Reply:
We do it after this has been reviewed and merged 👍